tests/TestCase/Cache/Engine/ApcEngineTest.php

```diff
@@ -34,7 +34,7 @@ class ApcEngineTest extends TestCase
     public function setUp()
     {
         parent::setUp();
-        $this->skipIf(!function_exists('apc_store'), 'Apc is not installed or configured properly.');
+        $this->skipIf(!function_exists('apcu_store'), 'Apc is not installed or configured properly.');
```

Comment:
@lorenzo Maybe need to change the message to say Apcu instead for easy debugging.